_index.md

```diff
@@ -23,25 +23,16 @@ Unless otherwise noted, everything in the repository is covered by the Apache Li
 
 ## Documentation Sections
 
-* [Project]({{< relref "doc/project" >}})
-  * How the OpenTitan project is organized
-  * Progress tracking
-* [User Guides]({{< relref "doc/ug" >}})
-  * How to get started with the repo
-  * How to emulate on an FPGA
-  * How hardware design is done in OpenTitan
-  * How verification is done in OpenTitan
-* [Reference Manuals]({{< relref "doc/rm" >}})
-  * Defining comportable IP peripherals
-  * Coding style guides for Verilog, Python, Hjson, C/C++ and Markdown
-  * OpenTitan tools
+* [General]({{< relref "doc" >}})
```

Comment:
Any suggestions for a better name than "General"?

Reply:
[General Documentation]
[Project wide documentation]
[Primary Documentation]

not in love with any of these either.